src/teleop/gui/src/components/ISHTask.vue

```diff
@@ -1,41 +1,275 @@
 <template>
-  <div>
-    <p>ISH</p>
-    <div class="box camera light-bg">
-      <Cameras :num-cams="4" :primary="false" />
+  <div class="wrapper">
+    <div class="box header">
+      <img
+        src="/static/mrover.png"
+        alt="MRover"
+        title="MRover"
+        width="48"
+        height="48"
+      />
+      <h1>ISH Dashboard</h1>
+      <div class="spacer"></div>
+      <!-- TODO: Add back comms indicators for ROSBridge Server Connection and Rover Connection -->
+      <!-- <div class="comms">
+      <ul id="vitals">
+        <li><LEDIndicator v-bind:connected="connections.websocket" name="Web Socket" /></li>
+        <li><LEDIndicator v-bind:connected="connections.lcm" name="Rover Connection Status" /></li>
+      </ul>
+    </div> -->
```

Comment:
Can we address this todo before merging

Reply:
spoke with joe offline, we didn't think there was a need for it this yr